### PR TITLE
Add 'since' version to new releases docs

### PR DESCRIPTION
**Pull request by dannify**
Missing monopackage version in rsp docs

### CODE REVIEW

File: packages/@react-spectrum/actionbar/docs/ActionBar.mdx [hunk 1]
```diff
@@ -40,7 +40,8 @@ keywords: [actionbar, selection, actions, collections, toolbar]
   componentNames={['ActionBar', 'ActionBarContainer']}
   sourceData={[
     {type: 'Spectrum', url: 'https://spectrum.adobe.com/page/action-bar/'}
-  ]} />
+  ]},
```
Comment (snowystinger) [inline packages/@react-spectrum/actionbar/docs/ActionBar.mdx:43]:
```suggestion
  ]}
```

File: packages/@react-spectrum/avatar/docs/Avatar.mdx [hunk 2]
```diff
@@ -31,7 +31,8 @@ keywords: [avatar]
 <HeaderInfo
   packageData={packageData}
   componentNames={['Avatar']}
-  sourceData={[{type: 'Spectrum', url: 'https://spectrum.adobe.com/page/avatar/'}]} />
+  sourceData={[{type: 'Spectrum', url: 'https://spectrum.adobe.com/page/avatar/'}]},
```
Comment (snowystinger) [inline packages/@react-spectrum/avatar/docs/Avatar.mdx:34]:
```suggestion
  sourceData={[{type: 'Spectrum', url: 'https://spectrum.adobe.com/page/avatar/'}]}
```

File: packages/@react-spectrum/tag/docs/TagGroup.mdx [hunk 3]
```diff
@@ -39,7 +39,8 @@ keywords: [TagGroup, status]
   componentNames={['TagGroup', 'Item']}
   sourceData={[
     {type: 'Spectrum', url: 'https://spectrum.adobe.com/page/tag/'}
-  ]} />
+  ]},
```
Comment (snowystinger) [inline packages/@react-spectrum/tag/docs/TagGroup.mdx:42]:
```suggestion
  ]}
```

**Comment by dannify** [comment]
Hahah I know how to code! It was a test :)

**Comment by snowystinger** [comment]
> Hahah I know how to code! It was a test :)

more worried about your standing as human linter now

**Review by snowystinger — APPROVED** [review]

**Review by devongovett — APPROVED** [review]

**Pull request merged**